tox.ini

```diff
@@ -0,0 +1,2 @@
+[pycodestyle]
+max-line-length = 5000
```

Comment:
CodeClimate checks (Python specific) do not pass without these changes so for other developers these are required.